dspace/config/modules/coar-notify-ldn.cfg

```diff
@@ -0,0 +1,12 @@
+#---------------------------------------------------------------#
+#---------------COAR NOTIFY LDN CONFIGURATION-------------------#
+#---------------------------------------------------------------#
+#     Configuration properties used  by Coar Notify and ldn     #
+#---------------------------------------------------------------#
+
+#Boolean to determine if Coar Notify is enabled globally for entire site.
+#default => true
+coar-notify.enabled = true
+
+# For debugging purposes only, skip the check on the IP range.
+#coar-notify.ip-range.enabled = false
```

Comment:
Because these configs are related to LDN, I'd recommend moving these all to the `ldn.cfg` file to keep everything together. You can just create a separate section in that file for the ones that are specific to COAR Notify.